modules/custom-post-types/comics/comics.css

```diff
@@ -8,9 +8,48 @@ body.dragging #jetpack-comic-drop-zone .dragging, body.uploading #jetpack-comic-
 	display: block;
 }
 
-body.uploading #jetpack-comic-drop-zone .uploading .spinner {
-	display: inline-block;
+body.uploading #jetpack-comic-drop-zone .uploading .spinner,
+body.uploading #jetpack-comic-drop-zone .uploading .spinner:after {
+	border-radius: 50%;
 	width: 60px;
+	height: 60px;
+}
+body.uploading #jetpack-comic-drop-zone .uploading .spinner {
+	display: block;
+	margin: 15px auto;
+	font-size: 10px;
+	position: relative;
+	text-indent: -9999em;
+	border-top: 1.1em solid rgba(255, 255, 255, 0.2);
+	border-right: 1.1em solid rgba(255, 255, 255, 0.2);
+	border-bottom: 1.1em solid rgba(255, 255, 255, 0.2);
+	border-left: 1.1em solid #ffffff;
+	-webkit-transform: translateZ(0);
+	-ms-transform: translateZ(0);
+	transform: translateZ(0);
+	-webkit-animation: load8 1.1s infinite linear;
+	animation: load8 1.1s infinite linear;
+}
+
+@-webkit-keyframes load8 {
+	0% {
+		-webkit-transform: rotate(0deg);
+		transform: rotate(0deg);
+	}
+	100% {
+		-webkit-transform: rotate(360deg);
+		transform: rotate(360deg);
+	}
+}
+@keyframes load8 {
+  0% {
+    -webkit-transform: rotate(0deg);
```

Comment:
Could you switch to tabs instead of spaces here, to make sure that the next person editing this file doesn't change all those spaces into tabs on save by mistake?